java/arcs/core/data/expression/PaxelParser.kt

```diff
@@ -204,15 +208,25 @@ object PaxelParser {
             )
         }
 
+    private val orderBySelectors =
+        (refinementExpression + many(comma + refinementExpression)).map { (o, r) ->
+            listOf(o) + r
+        }
+
+    private val orderByExpression = -token("orderby") +
+        (whitespace + orderBySelectors + optional(ows + token("descending"))).map {
```

Comment:
You should also be able to say "ascending" explicitly I think.

Reply:
Done